translate/src/modules/user/components/UserMenu.tsx

```diff
@@ -49,6 +65,39 @@ export function UserMenuDialog({
             </a>
           </li>
           <li className='horizontal-separator'></li>
+
+          <p className='help'>Choose appearance</p>
+          <span className='toggle-button'>
+            <button
+              type='button'
+              value='dark'
+              className={`dark ${theme === 'dark' ? 'active' : ''}`}
+              title='Use a dark theme'
+              onClick={() => handleThemeButtonClick('dark')}
+            >
+              <i className='icon far fa-moon'></i>Dark
+            </button>
```

Comment:
We should make the buttons and the label above localizable. Ideally, we should also factor out the button template.